generators/generator-conversational-core/README.md

```diff
@@ -1,30 +1,12 @@
-# generator-conversational-core [![NPM version][npm-image]][npm-url]
-> Yeoman generator for instantiating a bot template using the Conversational Core components.
+# Generator-conversational-core
+Yeoman generator for instantiating a bot template using the Conversational Core components.
 
-## Installation
+Bot framework Composer is configured to consume and instantiate a bot template using this Yeomen generator. 
 
-First, install [Yeoman](http://yeoman.io) and generator-conversational-core using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+The resulting bot template will have the following packages + routing logic to these packages.
 
-```bash
-npm install -g yo
-npm install -g generator-conversational-core
-```
-
-Then generate your new project:
-
-```bash
-yo conversational-core
-```
-
-## Getting To Know Yeoman
-
- * Yeoman has a heart of gold.
- * Yeoman is a person with feelings and opinions, but is very easy to work with.
- * Yeoman can be too opinionated at times but is easily convinced not to be.
- * Feel free to [learn more about Yeoman](http://yeoman.io/).
-
-## License
-Copyright (c) Microsoft Corporation. All rights reserved.
-
-[npm-image]: https://badge.fury.io/js/generator-conversational-core.svg
-[npm-url]: https://npmjs.org/package/generator-conversational-core
+# packages
```

Comment:
Hmm thats weird, I pushed the readMe changes to main but it shows that you are making this change